packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -905,4 +905,33 @@ whisperkit-cli transcribe --audio-path /path/to/audio.mp3
 # Or use your preferred model variant
 whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3 --verbose`,
 ];
+
+export const diffusionkit = (): string[] => [
```

Comment:
Can you move this section up in the file to preserve alphabetical order as much as possible? (just after `diffusers` for instance)